templates/nextflow-aurora-mysql.yaml

```diff
@@ -60,12 +60,31 @@ Parameters:
     Type: List<String>
     Description: List of admin IAM user and role ARNs (strings)
 
+  DbClusterMapping:
+    Type: String
+    Description: List of admin IAM user and role ARNs (strings)
+    AllowedValues:
+      - mysql5
+      - mysql8
+    Default: mysql5
+
+  KmsKeyAlias:
+    Type: String
+    Description: Alias name of KMS key to use for encryption
+    Default: alias/nextflow-aurora-bootstrap-lambda
+
 Mappings:
   ClusterSettings:
-    mysql:
+    mysql5:
       version: 5.7.mysql_aurora.2.07.1
       engine: aurora-mysql
       family: aurora-mysql5.7
+      enginemode: serverless
+    mysql8:
+      version: 8.0.mysql_aurora.3.04.0
+      engine: aurora-mysql
+      family: aurora-mysql8.0
+      enginemode: provisioned
```

Comment:
from the docs: The serverless engine mode only applies for Aurora Serverless v1 DB clusters. Aurora [Serverless v2 DB clusters use the provisioned engine mode](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbcluster.html#cfn-rds-dbcluster-enginemode).